library/cwm/src/lib/cwm/abstract_widget.rb

```diff
@@ -83,6 +83,15 @@ def self.widget_type=(type)
     # @!method help
     #   @return [String] translated help text for the widget
 
+    # @!method dynamic_help
+    #   Dynamic help is useful when content of help text is not static and when
+    #   help is recomputed with {Yast::CWM#ReplaceWidgetHelp}, then it can
+    #   return different result. Examples where it is useful is e.g. replace point which
+    #   have dynamic content and different help text depending on what is displayed.
+    #   Another example is CWM::Table which can have dynamic header depending on conditions.
+    #   In those cases after call of {Yast::CWM#ReplaceWidgetHelp}, it will return new help text
+    #   @return [String] translated help text for the widget.
```

Comment:
it is in fact same signature. Both are methods that return "string". Difference is that the first one is used to store its result into WidgetHash, but the second is passed as reference to WidgetHash. So in first case, later change of help content is not reflected, but the second one it is called repeatedly.